tests/example_project_configuration.py

```diff
@@ -26,6 +27,10 @@
         PydanticTimeSpine(
             node_relation=PydanticNodeRelation(alias="day_time_spine", schema_name="stuff"),
             primary_column=PydanticTimeSpinePrimaryColumn(name="ds_day", time_granularity=TimeGranularity.DAY),
+            custom_granularity_columns=[
+                PydanticTimeSpineCustomGranularityColumn(name="retail_year"),
+                PydanticTimeSpineCustomGranularityColumn(name="martian_week"),
```

Comment:
![ted lasso yes](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExaWpsY3o2ZnBvaDVqdDJvOGM1YXc1dDF5Znd2a2RwNThkbWp1YmNjYiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3WCNY2RhcmnwGbKbCi/giphy.gif)

Reply:
I wonder how long a martian week is, though. 🤔 

Wikipedia knows, of course..... https://en.wikipedia.org/wiki/Darian_calendar

Reply:
LMAO